src/components/Cell/Cell.tsx

```diff
@@ -166,6 +165,7 @@ export const Cell = ({
     <SimpleCell
       hasActive={hasActive}
       hasHover={hasActive}
+      {...(draggable ? { tabIndex: undefined } : {})}
```

Comment:
```suggestion
      tabIndex={draggable ? undefined : tabIndexProp}
```
чтобы не создавать объекты

Reply:
Тут нужно чтобы tabIndex который задается в Tappable не перебивался

Reply:
Ааа i see
Хм, тогда стоит в самом `Tappable` тоже сделать `tabIndex={isCustomElement && !props.disabled ? 0 : tabIndexProp}`, чтобы в дальнейшем не думать о том, то в Tappable не перебивалось

Reply:
@inomdzhon, в данном случае такое не подойдет, потому что мне нужно именно хардкодно сбросить `tabIndex`, иначе появится ненужная обводка у `Cell`

Reply:
Ааа i see #2)

Тогда, получается, если "завтра" кто-нибудь поправит в `Tappable` код на `tabIndex={isCustomElement && !props.disabled ? 0 : tabIndexProp}`, то тогда сломается ведь сброс? Это может, например, случается при рефактор `Tappable` (есть issue на эту тему)

Если поведение `draggable` де-факто конфликтует с `tabIndex`, то давайте тогда в `Tappable` передавать атрибут `draggable`, чтобы он при этом параметре не выставлял `tabIndex`

Reply:
Считаю что логика управления `tabIndex` должна быть за пределом `Tappable `. 

> если "завтра" кто-нибудь поправит в Tappable код...

То это будет мажорное изменение

Reply:
> Считаю что логика управления `tabIndex` должна быть за пределом `Tappable `.
> 
> > если "завтра" кто-нибудь поправит в Tappable код...
> 
> То это будет мажорное изменение

Будет мажорное изменение, но ты не поймёшь сломался ли `draggable` в `Cell` после этого